subsys/power/Kconfig

```diff
@@ -84,3 +84,8 @@ config PM_DEVICE_IDLE
 	  resumed based on the device usage even while the CPU or
 	  system is running.
 endmenu
+
+config HAS_PM
+	bool
+	help
+	  This option signifies that the target supports PM
```

Comment:
support PM in what way? SW? HW? 

I am still not convinced this is needed. In Zephyr, no boards should enable PM by default IMO, this will just cause mysterious failures with unit-testing and it will pull unncessary code when testing basic zephyr features on HW. Lets be selective about power tests and just enable those that are known to work.
with this solution, we will end up adding HAS_PM to almost each platform we have

Reply:
Not saying to enable PM by default, and @pabigot patch excluding certain targets from power tests is right. Though, we currently have just one condition to enable PM, that is `SYS_CLOCK_EXISTS`, but the subsystem *currently* depends on platform specific code.

Reply:
that option is just saying whether or not a target implements this code. It is not enabling power management at all.

Reply:
I agree with @nashif here: it will be a very rare system that can't support PM, probably only boards like nRF53 where there are two cores, and only one of them has the necessary pieces.  Maybe switch the logic to  `CONFIG_PM_UNSUPPORTED` defaulted to `n` so we only have to set it in the exceptional case.